lib/pengine/pe_output.c

```diff
@@ -1470,7 +1470,7 @@ failed_action_technical(pcmk__output_t *out, const xmlNode *xml_op,
     g_string_free(str, TRUE);
 }
 
-PCMK__OUTPUT_ARGS("failed-action", "xmlNodePtr", "uint32_t")
+PCMK__OUTPUT_ARGS("failed-action", "xmlNode *", "uint32_t")
```

Comment:
FYI, the main reason to avoid xmlNodePtr is that `const xmlNodePtr` does not work like `const xmlNode *`

Reply:
Yeah, `xmlNodePtr` is what's used in the libxml API headers --  rather than being quasi-internal, it's what they *advertise*. They use `xmlNode *` only when const is required.

I'm indifferent to which one we use for non-const. I've been using `xmlNodePtr` for a while because we were moving in that direction for a period of time, probably to match the API.

Personally, I don't understand why they created and advertised that typedef (and similar ones) in the first place... It just adds confusion IMO.